### DISCUSSION TITLE
discussion

### DISCUSSION THREAD

**Discussion by Vertacyl** [BEEmod/BEE2-items#3759] [General]
nobody has surprisingly said anything here ever so I'll just

**Comment by vrad-exe** [comment]
Hi! I thought about using this for questions and stuff, but we're actually considering releasing BEEmod on Steam at some point, so I figured it would be better to wait until then and use Steam Discussions. But this is good for now.

**Comment by Stnkybo** [comment]
I got ver 4.39.2 installed with and the auto portal stuff doesn’t seem to work. Idk if this is relevant but I have to 4.39.1 package installed and the patch can anyone help me out
